docs/dev-guide.rst

```diff
@@ -115,10 +113,9 @@ Hotness container starts after the start of containers running services required
 Makefile scripts that provide easier container management:
 
 * ``make up`` Starts all the container services
-* ``make restart`` Restarts all the container services that are either stopped or running 
+* ``make restart`` Restarts all the container services that are either stopped or running
 * ``make halt`` Stops and removes the containers
 * ``make bash`` Connects to hotness container
-* ``make logs`` Shows all logs of all containers
```

Comment:
Why is this removed?

Reply:
![2021-09-13_10-28](https://user-images.githubusercontent.com/39852038/133041758-cd43b7f8-611e-44ca-a656-79190a543a6d.png)

Because logs command behaves differently on podman compose. You need to specify 1 container to show logs.

But I think I can do some workarounds. I will provide this command back.